src/renderers/dom/server/__tests__/ReactServerRendering-test.js

```diff
@@ -45,33 +45,33 @@ describe('ReactServerRendering', function() {
       var response = ReactServerRendering.renderToString(
         <span>hello world</span>
       );
-      expect(response).toMatch(
+      expect(response).toMatch(new RegExp(
```

Comment:
`toMatch` used to silently convert the string into a RegExp and no longer does it. Replacing the few call sites we had to match the new behavior. It's IMO a good thing to have to pass a RegExp to toMatch :)